### PR TITLE
MONDO:0015429 & MONDO:0022737 choroideremia hypopituitarism [Merge]

### LINKED ISSUE #7553: MONDO:0015429 & MONDO:0022737 choroideremia hypopituitarism [Merge]
**Mondo term (ID and Label)**
MONDO:0015429 choroideremia-hypopituitarism syndrome

**Reason for deprecation**
The Orphanet ID on this is obsolete, the UMLS 'CN' id is not correct either. Seems like it could be conceptually similar to "choroideremia hypopituitarism" 

However, a Pubmed search of "choroideremia hypopituitarism" returns only 3 hits, one of which is the case report from 1989 PMID: 2624260 
PMID: 18978655 and PMID: 37732399 are also retrieved, but seem to focus on other disease concepts that are phenotypically similar (i.e. Oliver-McFarlane syndrome, MONDO:0010152) . Suggests that this may not be a valid condition in current literature?  Also noting that UMLS metathesaurus reports the SNOMEDCT concept on [715417002](http://identifiers.org/snomedct/715417002) is no longer active (obsoleted atom, looks like maybe retired in 2020?)

**Term to be merged with**
choroideremia hypopituitarism, http://purl.obolibrary.org/obo/MONDO_0022737
OR 
obsolete entirely with a suggested alternative term from Orphanet: "Instead, consider using [Xq21 microdeletion syndrome](https://www.orpha.net/en/disease/detail/1435) " 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by katiermullen**
closes #7553

@nicolevasilevsky could you please review this merge PR. Thank you!

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -480916,13 +480909,17 @@ name: choroideremia hypopituitarism
 def: "This is an X-linked recessive retinal degenerative disease that leads to degeneration of the choriocapillaris, the retinal pigment epithelium, and the photoreceptor of the eye. Hypopituitarism is the decreased (hypo) secretion of one or more of the eight hormones normally produced by the pituitary gland at the base of the brain." []
 subset: gard_rare {source="MONDO:GARD"}
 subset: rare
+synonym: "CHM-hypopituitarism syndrome" EXACT [Orphanet:1434]
 synonym: "choroideraemia co-occurrent with hypopituitarism" RELATED []
-synonym: "choroideraemia hypopituitarism" RELATED []
-synonym: "choroideremia co-occurrent with hypopituitarism" RELATED []
-synonym: "choroideremia hypopituitarism" EXACT []
+synonym: "choroideremia-hypopituitarism syndrome" EXACT [MONDO:0015429]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:480914]: this is an obsolete name from Orphanet, so I don't think we should add this as a synonym. I usually don't xrefs that point back to obsolete terms either, I add the xref from the original source (but since this term is obsolete, I don't think we should include it as a synonym).

Or we could include it with an xref to Orphanet and note it is a historic synonym that is no longer used (there is an annotation property for that- DEPRECATE, see: https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types)

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -480916,13 +480909,17 @@ name: choroideremia hypopituitarism
 def: "This is an X-linked recessive retinal degenerative disease that leads to degeneration of the choriocapillaris, the retinal pigment epithelium, and the photoreceptor of the eye. Hypopituitarism is the decreased (hypo) secretion of one or more of the eight hormones normally produced by the pituitary gland at the base of the brain." []
 subset: gard_rare {source="MONDO:GARD"}
 subset: rare
+synonym: "CHM-hypopituitarism syndrome" EXACT [Orphanet:1434]
 synonym: "choroideraemia co-occurrent with hypopituitarism" RELATED []
-synonym: "choroideraemia hypopituitarism" RELATED []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:480920]: why did you remove these synonyms here?

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
see inline comments

**Comment by nicolevasilevsky** [comment]
@katiermullen feel free to reassign this back to me when it is ready for re-review

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:480914]: removed this synonym

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:480920]: mistake. added back.

**Comment by katiermullen** [comment]
@nicolevasilevsky please re-review :)

Thank you!

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:480920]: all good!

**Review by nicolevasilevsky — APPROVED** [review]
Looks great, @katiermullen, thank you!
